apisix/admin/plugins.lua

```diff
@@ -23,6 +23,7 @@ local pcall     = pcall
 local table_sort = table.sort
 local table_insert = table.insert
 local get_uri_args = ngx.req.get_uri_args
+local v1 = require("apisix.control.v1")
```

Comment:
It would be better to use `plugin.get_all` directly, instead of let admin API depends on control API. And the fields returned by this one is different from the control API one?

Reply:
yes, different, at first, this api was planned to return only the schema of the plugins, not included as `route`, `service`